client/src/app/site/pages/meetings/pages/motions/services/export/motion-xlsx-export.service/motion-xlsx-export.service.ts

```diff
@@ -187,11 +187,11 @@ export class MotionXlsxExportService {
                 data.push(
                     ...commentIds.map(commentId => {
                         if (commentId === PERSONAL_NOTE_ID) {
-                            return motion && motion.getPersonalNote()! && motion.getPersonalNote()!.note;
+                            return motion?.getPersonalNote()?.note ?? ``;
                         } else {
                             const section = this.commentRepo.getViewModel(commentId)!;
                             const motionComment = motion.getCommentForSection(section);
-                            return motionComment && motionComment.comment
+                            return  motionComment?.comment
```

Comment:
There seems to be something wrong with the linter
```suggestion
                            return motionComment?.comment
```